6-System-Maintenance/6.2.4.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+awk -F: '($1!~/(halt|sync|shutdown|nfsnobody)/ &&
+$7!~/^(\/usr)?\/sbin\/nologin(\/)?$/ && $7!~/(\/usr)?\/bin\/false(\/)?$/) {print $1 " " $6 }' /etc/passwd | while read -r user dir; do
```

Comment:
I think this belongs at the end of line 3. Keep in mind to put a space between the `&&` and the `$7!...`